cluster-autoscaler/utils/scheduler/scheduler.go

```diff
@@ -106,3 +117,31 @@ func ResourceToResourceList(r *schedulerframework.Resource) apiv1.ResourceList {
 	}
 	return result
 }
+
+// GetSchedulerConfig loads scheduler config from a path.
+func GetSchedulerConfig(path string) (*scheduler_config.KubeSchedulerConfiguration, error) {
```

Comment:
nit: `ConfigFromPath`? (So it becomes `scheduler.ConfigFromPath(path)` outside of this package.)

Reply:
Thanks for the suggestion. I have changed the name to `ConfigFromPath`. `scheduler.ConfigFromPath` feels less vague than `scheduler.GetSchedulerConfig`.